reark/src/androidTest/java/io/reark/reark/data/stores/ContentProviderStoreTest.java

```diff
@@ -115,68 +117,71 @@ public void getOnce_WithNoData_ReturnsNoneValue_AndCompletes() {
-        // want to make sure the provider stores can return a larger listing of results.
-        store.getAllOnce(0).subscribe(testSubscriber);
+        store.getOnceAndStream(TestStore.getIdFor("bacon")).subscribe(testSubscriber);
 
         // ASSERT
-        testSubscriber.awaitTerminalEvent();
-        testSubscriber.assertCompleted();
+        testSubscriber.awaitTerminalEvent(50, TimeUnit.MILLISECONDS);
+        testSubscriber.assertNotCompleted();
         testSubscriber.assertNoErrors();
         testSubscriber.assertReceivedOnNext(expected);
     }
 
+    // The following tests are not part of the public API, but rather test what
+    // the content provider core exposes for extending classes.
+
     @Test
-    public void getOnceAndStream_WithData_ReturnsData_AndDoesNotComplete() {
+    public void core_GetAllOnce_WithData_ReturnsData_AndCompletes() {
         // ARRANGE
-        TestSubscriber<String> testSubscriber = new TestSubscriber<>();
-        List<String> expected = Collections.singletonList("spinach");
+        TestSubscriber<List<String>> testSubscriber = new TestSubscriber<>();
+        List<List<String>> expected = Collections.singletonList(Collections.singletonList("parsnip"));
 
         // ACT
-        store.getOnceAndStream(TestStore.getIdFor("spinach")).subscribe(testSubscriber);
+        core.getAllCached(TestStore.getIdFor("parsnip")).subscribe(testSubscriber);
 
         // ASSERT
-        testSubscriber.awaitTerminalEvent(50, TimeUnit.MILLISECONDS);
-        testSubscriber.assertNotCompleted();
+        testSubscriber.awaitTerminalEvent();
+        testSubscriber.assertCompleted();
         testSubscriber.assertNoErrors();
         testSubscriber.assertReceivedOnNext(expected);
     }
 
     @Test
-    public void getOnceAndStream_WithNoData_ReturnsNoneValue_AndDoesNotComplete() {
+    public void core_GetAllOnce_WithWildcardQuery_WithData_ReturnsAllData_AndCompletes() {
         // ARRANGE
-        TestSubscriber<String> testSubscriber = new TestSubscriber<>();
-        List<String> expected = Collections.singletonList(NONE);
+        TestSubscriber<List<String>> testSubscriber = new TestSubscriber<>();
+        List<List<String>> expected = Collections.singletonList(Arrays.asList("parsnip", "lettuce", "spinach"));
 
         // ACT
-        store.getOnceAndStream(TestStore.getIdFor("bacon")).subscribe(testSubscriber);
+        // Wildcard depends on content provider. For tests we just use 0 while on SQL backend
+        // this would be an asterisk. The exact wildcard is not important for the test as we just
+        // want to make sure the provider stores can return a larger listing of results.
+        core.getAllCached(0).subscribe(testSubscriber);
 
         // ASSERT
-        testSubscriber.awaitTerminalEvent(50, TimeUnit.MILLISECONDS);
-        testSubscriber.assertNotCompleted();
+        testSubscriber.awaitTerminalEvent();
+        testSubscriber.assertCompleted();
```

Comment:
This is not needed as you have `awaitTerminalEvent` and `assertNoErrors` unless you want to have it for clarity